### PR TITLE
Golang implementation

### PR DESCRIPTION
**Pull request by DazWilkin**
Addresses #11

### CODE REVIEW

**Comment by DazWilkin** [comment]
I noticed a mistake; I'll fix it today.

File: golang/README.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+```bash
+ENDPOINT=$(\
+  gcloud run services list \
+  --project=${GCP_PROJECT} \
+  --region=${GCP_REGION} \
+  --platform=managed \
+  --format="value(status.address.url)" \
+  --filter="metadata.name=grpc-calculator") 
+ENDPOINT=${ENDPOINT#https://} && echo ${ENDPOINT}
```
Comment (gnossen) [inline golang/README.md]: Nice! :+1:

Comment (gnossen) [reply golang/README.md]: Nit: It would be nice to have this in the other two `README` files as well.

Comment (DazWilkin) [reply golang/README.md]: I've created a branch in my repo with the changes for the other languages

**Comment by gnossen** [comment]
@DazWilkin Looks like there's a conflict that needs to be resolved in `deploy.sh`. Otherwise, this looks good to go. :+1:

**Comment by DazWilkin** [comment]
@gnossen it's a consistency recommendation.

I'm unable to 'Resolve conflicts' but you may be able to do so.... 

The change ensures all flags are `--FLAG=VALUE` rather than some `--FLAG VALUE` and some `--FLAG=VALUE`

**Comment by gnossen** [comment]
FYI, in the future, you can resolve yourself with
```sh
git fetch origin
git merge origin/master
```

**Pull request merged**